metadata_backend/api/middlewares.py

```diff
@@ -50,6 +53,78 @@ async def http_error_handler(req: Request, handler: Callable) -> Response:
     return http_error_handler
 
 
+def jwt_middleware() -> Callable:
```

Comment:
We can just import `jwt_authentication` and add `middlewares=[jtw_authentication ]` in server.py, no need to wrap it inside another function. (This actually applies to error middleware as well).

Reply:
Upon checking the aiohttp documentation, I now recognize this is a good point. I guess the original idea with this  was that there might be multiple methods under the `error_middleware()` so that `server = web.Application(middlewares=[error_middleware()]` in `server.py`, would access them all. That never ended being the case, so good catch!